section-5/exercises/denysKozlovskyiSection5.js

```diff
@@ -0,0 +1,170 @@
+// Exercise 1:
+// 1. Write a function which will check if a user has a valid ticket ID and does user have 18+ to enter a club.
+// If condition is true then print 'User is allowed to enter the club!' if false then print 'User entrance is not allowed!'
+// Hint: use operator *&&* to check both conditions.
+function checkingUserIdTicket(ticketID, userAge){
+    if (ticketID === 'Yes' && userAge >=18){
+        return console.log("User entrance is allowed!");
```

Comment:
console.log just print the message in console and does not return anything. So in this case return is not needed. You can return simple string if you would like to return somethings from function.